controllers/comments.go

```diff
@@ -23,20 +23,31 @@ func NewCommentController(db databases.Database, logger *zap.Logger) *commentCon
 	}
 }
 
+func (cc *commentController) LogInfo(message string) {
+	if cc.logger != nil {
+		cc.logger.Info(message)
+	}
+}
+
 func (cc *commentController) CreateComment(ctx echo.Context) error {
 	var comment models.Comment
-	user_id, err := tools.ExtractUserID(ctx)
+	userID, err := tools.ExtractUserID(ctx)
 	if err != nil {
 		cc.logger.Error(err.Error())
-		return ctx.JSON(http.StatusUnauthorized, echo.Map{"error": "there is problem with your user"})
+		return ctx.JSON(http.StatusUnauthorized, echo.Map{"error": "there is a problem with your user"})
 	}
+
 	if err := ctx.Bind(&comment); err != nil {
+		cc.LogInfo("Unable to parse comment")
```

Comment:
Use `log.Gl.Error(err.Error())` instead and you are all done here, no need for a custom (and vague) message.